driver-core/src/main/com/mongodb/internal/connection/ZstdCompressor.java

```diff
@@ -27,6 +27,14 @@
 import java.util.List;
 
 class ZstdCompressor extends Compressor {
+    ZstdCompressor() {
+        try {
+            Class.forName("com.github.luben.zstd.Zstd");
+        } catch (ClassNotFoundException e) {
+            throw new RuntimeException(e);
+        }
```

Comment:
Following an offline discussion with @hisener: maybe adding a comment and exception message can clarify this code somewhat:
```suggestion
        try {
            // Trigger static class initialization so that associated blocking operations don't
            // happen on `#compress`-invoking threads.
            Class.forName("com.github.luben.zstd.Zstd");
        } catch (ClassNotFoundException e) {
            throw new RuntimeException("Failed to initialize Zstd", e);
        }
```

(We also discussed alternative approaches, like introducing an expression such as `Zstd.class.getName()` or a statement such as `Class<?> clazz = Zstd.class;`, but the current `Class.forName` approach, albeit more verbose, seems more idiomatic.)